lib/mixpanel/context.tsx

```diff
@@ -62,13 +68,12 @@ export function MixpanelProvider({ children, trackingService, defaultEventContex
   );
 
   useEffect(() => {
-    // Only run on Web / Client
-    if (typeof window === 'undefined') {
+    if (disableSessionStorage) {
```

Comment:
Hoe ga je nu op mobile de utm functionaliteit oplossen? Als dat helemaal vervalt op mobile zou het beter disableUTM kunnen heten.

Reply:
Ja, UTM kunnen we uiteindelijk wel gebruiken via Deeplinking dus vandaar dat ik het zo noem.